users/models.py

```diff
@@ -1,3 +1,19 @@
 from django.db import models
 
-# Create your models here.
+
+class Users(models.Model):
+    first_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30)
+    email = models.EmailField(max_length=70, unique=True)
+    phone_number = models.CharField(max_length=13, unique=True)
+    passport_number = models.CharField(max_length=8, unique=True)
+    identity_code = models.CharField(max_length=15, unique=True)
+    company = models.ForeignKey("self", on_delete=models.CASCADE)
```

Comment:
does this mean that when we delete a company - we delete all users? we discussed that it should not be like that. Instead, use on_delete=DO_NOTHING